### PR TITLE
Slovenian language - translation added

### PR DESCRIPTION
**Pull request by sebastjanm**
Slovenian translation strings for Opencast studio

### CODE REVIEW

File: src/languages.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const languages = [
   { short: 'fa', long: 'Farsi', rtl: true },
   { short: 'fr', long: 'Français', rtl: false },
   { short: 'tr', long: 'Turkish', rtl: false },
+  { short: 'sl', long: 'Slovenščina', rtl: false },
```
Comment (LukasKalbertodt) [inline src/languages.js:10]: This list is sorted alphabetically, except for English (which is first). So please swap Turkish and Slovenščina :)

File: src/i18n/locales/sl.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+
+  "confirm-discard-recordings": "Ali ste prepričani, da želite zavreči posnetke? Tega ni mogoče razveljaviti!",
+  "confirm-cancel-recording": "Trenutno snemate! Ali ste prepričani, da želite preklicati in s tem zavreči snemanje?",
+
+  "sources-video-display-selected": "Zaslon je izbran",
+  "sources-video-display-and-user-selected": "Zaslon & video kamera izbrana",
+  "sources-video-user-selected": "Video kamera je izbrana",
+  "sources-video-question": "Izberite željeni video vir za snemanje",
+  "sources-video-none-available": "Vaš brskalnik / naprava ne podpira zajema zaslona ali video kamere:-(",
```
Comment (LukasKalbertodt) [inline src/i18n/locales/sl.json:47]: I think there is missing a space before ":-(". Or does Slovenian not put a space there?

File: src/i18n/locales/sl.json [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  "error-lost-audio-stream": "Snemanje zvoka se je nepričakovano končalo! Ali se je prekinila povezava s mikrofonom ali ste po nesreči zamenjali avdio vir??",
+  "error-lost-stream-end-recording": "Video ali avdio snemanje se je nepričakovano končalo in zaradi tega je bilo aktivno snemanje predčasno zaustavljeno! Ali se je prekinila povezava s video kamero ali mikrofonom? Ali ste po naključju zamenjali kamero ali avdio povezavo oziroma preklicali potrebna dovoljenja?",
+
+  "review-heading": "Ali si zadovoljen z video posnetkom?",
+  "review-error-empty-recording": "Napaka: vaš posnetek je popolnoma prazen. Ste mogoče snemanje ustavili skoraj takoj po tem, ko ste ga začeli? Poleg tega se prepričajte, da je vaš sistem dovolj zmogljiv za snemanje videa. (to morda ne velja, če druga aplikacija močno obrremenjuje vaš sistem ali pa če je vaša računalnik zelo star). V primeru  da je vaš sistem dovolj zmogljiiv za viideo snemanje, ter snemali več kot nekaj sekund in še vedno vidite to napako: Prosimo prijavite to kot napako na GitHubu.",
```
Comment (LukasKalbertodt) [inline src/i18n/locales/sl.json:75]: Is "GitHubu" (at the very end of this line) a typo? If not, that's very cute :D

File: src/i18n/locales/sl.json [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+  "upload-time-an-hour": "ura",
+  "upload-time-seconds": "sekunde",
+  "upload-time-minutes": "minute",
+  "upload-time-hours": "ure",
+  "upload-time-left": "{{time}} ostane",
+
+  "warning-https": "Večina brskalnikov ne dovoljuje zajema video posnetkov prek nešifrirane povezave (HTTP). Preklopite na HTTPS.",
+  "warning-recorder-not-supported": "Vaš brskalnik ne podpira snemanja video posnetkov.",
+  "warning-recorder-safari-hint": "Če uporabljate Safari, lahko v nastavitvah omogočite eksperimentalno funkcijo »MediaRecorder«. V Apple MacOS-u priporočamo prehod na Chrome ali Firefox.",
+  "warning-missing-connection-settings": "Povezava z Opencastom ni v celoti vzpostavljena: nalaganje je onemogočeno. Konfigurirajte povezavo v <1> nastavitvah </1> (posnetka ne boste izgubili).",
```
Comment (LukasKalbertodt) [inline src/i18n/locales/sl.json:103]: `<1> nastavitvah </1>` should be `<1>nastavitvah</1>` (without space on the inside)

File: src/i18n/locales/sl.json (hunk 3 above)
Comment (sebastjanm) [reply src/i18n/locales/sl.json:75]: Thx :) 
This is correct. In Slovenian language we usually don't  use "-" for declension of nouns.

File: src/i18n/locales/sl.json (hunk 4 above)
Comment (sebastjanm) [reply src/i18n/locales/sl.json:103]:
```suggestion
  "warning-missing-connection-settings": "Povezava z Opencastom ni v celoti vzpostavljena: nalaganje je onemogočeno. Konfigurirajte povezavo v <1>nastavitvah</1> (posnetka ne boste izgubili).",
```

File: src/languages.js (hunk 1 above)
Comment (sebastjanm) [reply src/languages.js:10]: @LukasKalbertodt .. how can i do this?  New pull request?

**Comment by sebastjanm** [comment]
- > Are there some Slovenian users of Opencast Studio? Some universities?
Not yet, but 'm doing my best to promote Opencast Studio, since it is awesome idea. Keep up the good work.

**Comment by LukasKalbertodt** [comment]
>  .. how can i do this? New pull request?

You can just modify the file (locally on your PC), commit the changes and push. Then the new commit will automatically appear here. If you need more help with git, just tell me. It's just really difficult to judge the git expertise of others on the internet and explaining everything in great detail might insult some people who know that stuff already. 

> Not yet, but 'm doing my best to promote Opencast Studio, since it is awesome idea.
Keep up the good work.

Great, thanks a bunch!

**Comment by sebastjanm** [comment]
@LukasKalbertodt Yes please merge and i will complete later

**Comment by LukasKalbertodt** [comment]
I fixed the few comments I left. As I couldn't add commits to this PR, I created a new one: #656. It includes your commit and is merged now. Again, thanks for your translation! Whenever you find the time to complete the translation, just create a PR or let me know so that I can tell you which translations are missing, exactly.

**Pull request merged**